test/stablex/stablecoin_converter.js

```diff
@@ -1253,6 +1307,11 @@ contract("StablecoinConverter", async (accounts) => {
     })
   })
   describe("getEncodedAuctionElements()", async () => {
+    it("returns null when no orders.", async () => {
```

Comment:
<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia3.giphy.com%2Fmedia%2FSIXreftELj2N2%2Fgiphy.gif&token=unrelated" data-gitmeme-token="unrelated"><img src="https://media3.giphy.com/media/SIXreftELj2N2/giphy.gif" title="Created by gitme.me with /unrelated"/></a>

Reply:
I will remove this change.